ES/ESdd041.xml

```diff
@@ -353,16 +346,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <keywords scheme="#ethioauthlist">
                <term key="MonasticLiterature"/>
                <term key="RitualsAndRites"/>
+               <term key="Liturgy"/>
+               <term key="Hagiography"/>
```

Comment:
It is not "Hagiography", look for "Rituals", "MonasticLiterature".

Reply:
Sure! I wanted to click "historiography"!